app/src/main/AndroidManifest.xml

```diff
@@ -70,8 +70,8 @@
     <activity
       android:name=".settings.profile.ProfileRenameActivity" />
     <activity
-      android:name=".settings.profile.ProfileResetPinActivity"
-      android:screenOrientation="portrait" />
+      android:name=".settings.profile.ProfileResetPinActivity">
+    </activity>
```

Comment:
Nit update:
`<activity android:name=".settings.profile.ProfileResetPinActivity" />`

Reply:
@rt4914 You said to fix 1 and 3, but told that we will add test cases later, so should I check previously written test cases?

Reply:
No actually, you should follow the steps mentioned in 1 and 3. Our final aim is to make sure that if user has typed something and does orientation change in between than that text should not get removed.

Reply:
@rt4914 Not able to figure it out, What I am thinking is that I need to look at the view model for profile reset pin activity/presenter and make some changes for a configuration change. Can you provide me a hint?

Reply:
@gunishjain You need to make sure that if user enter's some content before doing orientation change, then after orientation change that content should be present in the edit text.